### PR TITLE
116 zenodo url

### PR DESCRIPTION
**Pull request by sblack-usu**

### CODE REVIEW

**Review by Maurier — CHANGES_REQUESTED** [review]
Minor suggestion and a question

File: management/refresh_submission_url_zenodo.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+import motor
+import json
+from beanie import init_beanie
+
+from dspback.config import get_settings
+from dspback.pydantic_schemas import Submission
+
+'''
+This python script updates the ECL submission urls.
```
[the remark this answers is not in the document]
Comment (sblack-usu) [reply management/refresh_submission_url_zenodo.py]: yep

File: tests/test_records.py [hunk 2]
```diff
@@ -24,7 +24,7 @@ async def test_hydroshare_to_submission(hydroshare):
 
 async def test_zenodo_to_submission(zenodo):
     zenodo_record = ZenodoRecord(**zenodo)
-    zenodo_record.publication_date = None
+    assert zenodo_record.state == "unsubmitted"
```
[the remark this answers is not in the document]
Comment (sblack-usu) [reply tests/test_records.py]: We aren't using state anymore

**Review by Maurier — APPROVED** [review]
Looks good. I tried the script and all seems to work as intended.

**Pull request merged**